coop_shift/model/shift_shift.py

```diff
@@ -394,15 +374,12 @@ def _compute_end_date_fields(self):
                 "You can not create Shift if your timezone is not defined."))
         for shift in self:
             if shift.date_end:
-                shift.end_date_for_mail = datetime.strftime(
-                    fields.Datetime.from_string(shift.date_end),
-                    "%d/%m/%Y %H:%M")
+                shift.end_date_for_mail = \
+                    datetime.strftime(shift.date_end, "%d/%m/%Y %H:%M")
```

Comment:
What's this field for? `end_date_for_mail`
Looks like it's a formatted string used in templates, right?

In Odoo 12 we can just do this on templates, and it works as expected, using the user timezone/locale:

```html
<!-- For dates; -->
${format_date(object.date_end)}

<!-- or datetime -->
${format_datetime(object.date_end)}

<!-- This is also possible, and the result will be the same than previous code (datetime without seconds) -->
<!-- Although I wouldn't go for it because it ignores locale datetime format -->
${format_datetime(object.date_end, dt_format="dd/MM/YYYY HH:mm")}
```

Please see if you can remove it

Reply:
@ivantodorovich : this field not used in email template at lest not in coop_shift module